groups/rds/security_groups.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_security_group" "db_security_group" {
+  name        = "${var.environment}-${var.service}-rds"
+  description = "Allow TLS inbound traffic"
+  vpc_id      = data.aws_vpc.placement.id
+
+  ingress {
+    description      = "Ingress from permitted CIDRs"
+    from_port        = 3306
+    to_port          = 3306
+    protocol         = "tcp"
+    cidr_blocks      = concat(local.placement_subnet_cidrs, local.automation_subnet_cidrs)
```

Comment:
I'd move this concatenation into the locals file. That way we can do all our manipulation in one place and it makes the definition here easier to read. For example [here](https://github.com/companieshouse/kafka-stack/blob/main/groups/kafka/locals.tf#L12-L17) and [here](https://github.com/companieshouse/terraform-modules/blob/main/aws/kafka/security-groups.tf#L11). It also then gives us a common approach for modules